test/EFCore.PG.FunctionalTests/MigrationsInfrastructureNpgsqlTest.cs

```diff
@@ -215,6 +217,21 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
         [ConditionalFact(Skip = "Implement")]
         public override void Can_diff_against_2_1_ASP_NET_Identity_model() {}
+
+        public class MigrationsInfrastructureNpgsqlFixture : MigrationsInfrastructureFixtureBase
+        {
+            protected override ITestStoreFactory TestStoreFactory => NpgsqlTestStoreFactory.Instance;
+
+            public override MigrationsContext CreateContext()
+            {
+                var options = AddOptions(
+                        new DbContextOptionsBuilder()
+                            .UseNpgsql(TestStore.ConnectionString, b => b.ApplyConfiguration().CommandTimeout(NpgsqlTestStore.CommandTimeout)))
+                    .UseInternalServiceProvider(ServiceProvider)
+                    .Options;
+                return new MigrationsContext(options);
+            }
```

Comment:
This can be rewritten as an expression bodies method.

Reply:
Yeah I know, but this boilerplate is a pure copy-paste from EF so I'd rather leave it like it is upstream (for easier diffing/syncing).